agent/src/agent/cli/pipeline.py

```diff
@@ -339,14 +340,31 @@ def export(dir_path):
     """
     Export pipelines' config into file
     """
+
+    def _export_file(path, pipeline_name):
+        destination = os.path.join(os.path.dirname(dir_path), 'processing_files', pipeline_name)
```

Comment:
the file name should be the same as original value, destination path is the same dir